src/modules/launcher/Wox.Plugin/Common/VirtualDesktopHelper.cs

```diff
@@ -0,0 +1,411 @@
+﻿// Copyright (c) Microsoft Corporation
+// The Microsoft Corporation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using System;
+using System.Collections.Generic;
+using System.Runtime.InteropServices;
+using Microsoft.Win32;
+using Wox.Plugin.Common.Win32;
+using Wox.Plugin.Logger;
+
+namespace Wox.Plugin.Common.VirtualDesktop
+{
+    /// <summary>
+    /// Helper class to work with Virtual Desktops.
+    /// This helper uses only public available and documented COM-Interfaces or informations from registry.
+    /// </summary>
+    /// <remarks>
+    /// To use this hleper you have to create an instance of it and acces the method via the helper instance.
+    /// We are only allowed to use public documented com interfaces.
+    /// </remarks>
+    /// <seealso href="https://docs.microsoft.com/en-us/windows/win32/api/shobjidl_core/nn-shobjidl_core-ivirtualdesktopmanager">Documentation of IVirtualDesktopManager interface</seealso>
+    /// <seealso href="https://docs.microsoft.com/en-us/archive/blogs/winsdk/virtual-desktop-switching-in-windows-10">CSharp example code for IVirtualDesktopManager</seealso>
+    public class VirtualDesktopHelper
+    {
+        /// <summary>
+        /// Value indicating if Virtual Desktop Manager is successfull initialized
+        /// </summary>
+        private readonly bool _vdmInitSuccessfully;
```

Comment:
ToDo: Create public method to access information or return false/true from `VirtualDesktopHelper()` method.